integrations/symfony-bundle.rst

```diff
@@ -1,74 +1,60 @@
 Symfony Bundle
 ==============
 
-The usage documentation is split into two parts. First we explain how to configure the bundle in an application. The second part is for developing reusable Symfony bundles that depend on an HTTP client defined by the Httplug interface.
+This bundle integrate HTTPlug with the Symfony framework. The bundle helps to register services for all your clients and makes sure all the configuration is in one place. The bundle also feature a toolbar plugin with information about your requests.
 
-For information how to write applications with the services provided by this bundle, have a look at the [Httplug documentation](http://docs.php-http.org).
+Usage
+`````
 
+.. code-block:: yaml
 
-Use in Applications
--------------------
+    httplug:
+        plugins:
+            logger: ~
+        clients:
+            acme:
+                factory: 'httplug.factory.guzzle6'
+                plugins: ['httplug.plugin.logger']
+                config:
```

Comment:
if we have nothing in config, we should omit this line. or add the no ssl very thing or timeout, if we want to illustrate its possible to configure guzzle.